var/spack/repos/builtin/packages/fparser/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Fparser(CMakePackage):
+    """This C++ library offers a class which can be used to parse and
+    evaluate a mathematical function from a string (which might be eg
```

Comment:
Can we have a short one-line description first and a longer description as a separate paragraph:
```suggestion
    """Parse and evaluate a mathematical function from a string.
    
    This C++ library offers a class which can be used to parse and
    evaluate a mathematical function from a string (which might be eg
```
?